mapit/shortcuts.py

```diff
@@ -78,7 +78,14 @@ def output_json(out, code=200):
 def output_polygon(content_type, output):
     response = http.HttpResponse(content_type='%s; charset=utf-8' % content_type)
     response['Access-Control-Allow-Origin'] = '*'
-    response['Cache-Control'] = 'max-age=2419200'  # 4 weeks
+
+    # HACK: Some polygons are too large to store in memcached.
+    # PyLibMC throws an exception when this happens and the request fails.
+    # Setting max-age to 0 to prevent Django trying to cache in memcached
+    # and s-maxage so any 'upstream' caching can still be attempted.
+    # 2419200 is 4 weeks.
+    response['Cache-Control'] = 's-maxage=2419200, max-age=0,'
```

Comment:
I'm sure it probably doesn't matter, but I don't think I've seen a header with a final comma anywhere.